examples/monolithic_build/README.md

```diff
@@ -1,17 +1,50 @@
 [//]: # (SPDX-License-Identifier: CC-BY-4.0)
 
-# Single-level mldsa-native in a single compilation unit
+# Monolithic Build (C Backend)
 
-This directory contains a minimal example for how to build a single instance of mldsa-native in a single compilation
-unit. Only the C-backend is exercised.
+This directory contains a minimal example for building mldsa-native for a single
+parameter set of ML-KEM as a single compilation unit using the auto-generated
```

Comment:
Sorry for not catching earlier, still an ML-KEM reference here.

Reply:
Fixed.